modules/storages/app/common/storages/peripherals/nextcloud_connection_validator.rb

```diff
@@ -59,6 +60,17 @@ def query
                      .call(storage: @storage, auth_strategy:)
       end
 
+      def files
+        @files ||= Peripherals::Registry
+                     .resolve("#{@storage.short_provider_type}.queries.files")
+                     .call(
+                       storage: @storage,
+                       auth_strategy: Peripherals::Registry
```

Comment:
🟡 also minor, can we extract the userless auth strategy to a private method, too.

naming the current `noop`, and this one `userless`. IMHO a small improvement of readability